### DISCUSSION TITLE
EMP has no real use

### DISCUSSION THREAD

**Discussion by ViktorShield** [FakeFishGames/Barotrauma#9177] [Ideas and suggestions]
Creatures that get affected by EMP grenades and/or weaponry. Maybe some alien sub deconstruction scavenging bots. Reference:
![image](https://user-images.githubusercontent.com/84222807/169438154-c165cc01-49db-4450-bbce-bb319389c73c.png)
On the other hand creatures creating EMP blasts would be a good addition as well. Piezo-jellyfish?

**Comment by gerakis** [comment]
There is a piezo jellyfish. Its called a black moloch.

Otherwise I agree emp has no real use. Sort of like oxygenite.

**Reply by LeakingAmps** [reply]
Oxygenite is *very* useful once you have enough - load it into your (nuclear) railgun shells for extra spicy rounds. Very good against pirates (not sure those even exist in the current version - I've gone a weirdly long time without seeing any), and armored critters.
